.github/workflows/upload-release-to-plugin-repo.yml

```diff
@@ -0,0 +1,65 @@
+on:
+  release:
+    types: [released]
+
+name: Upload Gutenberg plugin to WordPress.org plugin repo
+
+jobs:
+  upload:
+    name: Upload Gutenberg Plugin
+    runs-on: ubuntu-latest
+    env:
+      PLUGIN_REPO_URL: 'https://plugins.svn.wordpress.org/gutenberg'
+      STABLE_TAG_REGEX: 'Stable tag: [0-9]\+\.[0-9]\+\.[0-9]\+\s*'
+      SVN_USERNAME: ${{ secrets.svn_username }}
+      SVN_PASSWORD: ${{ secrets.svn_password }}
+      VERSION: ${{ github.event.release.name }}
+    steps:
+      - name: Check out Gutenberg trunk from WP.org plugin repo
+        run: svn checkout "$PLUGIN_REPO_URL/trunk"
+
+      - name: Get previous stable tag
+        id: get_previous_stable_tag
+        run: echo ::set-output name=stable_tag::$(grep "$STABLE_TAG_REGEX" ./trunk/readme.txt)
+
+      - name: Delete everything
+        working-directory: ./trunk
+        run: find . -maxdepth 1 -not -name ".svn" -not -name "." -not -name ".." -exec rm -rf {} +
+
+      - name: Download and unzip Gutenberg plugin asset into trunk folder
+        env:
+          PLUGIN_URL: ${{ github.event.release.assets[0].browser_download_url }}
+        run: |
+          curl -L -o gutenberg.zip $PLUGIN_URL
+          unzip gutenberg.zip -d trunk
+          rm gutenberg.zip
+
+      - name: Replace the stable tag placeholder with the existing stable tag on the SVN repository
```

Comment:
Not sure I understand why this step is needed? Do we do that in the release script as well? I mean  we're supposed to update the stable tag after the svn tag is created, right?

Reply:
> Not sure I understand why this step is needed? Do we do that in the release script as well? I mean we're supposed to update the stable tag after the svn tag is created, right?

Yeah, I carried this one over from the release script, and it didn't seem to make too much sense to me either :grimacing: 
Happy to remove it :+1:

Reply:
Well if it's in the release script, maybe @jorgefilipecosta know why it's needed.

Reply:
The readme.txt of the plugin is maintained on the Github repository. But it does not refer a stable tag, it contains "Stable tag: V.V.V" as the stable tag.
When we commit something to trunk we first need to put the previous stable tag on the placeholder. Then when tagging the release we update the stable tag again.

Reply:
Okay, I think I understand now. I'll rephrase, to highlight what I think is crucial here:

1. We update trunk to match the build we've just produced.
2. We tag that snapshot of trunk with the current version.
3. We set the stable version in `readme.txt` to point to that tag.

Steps 1. and 3. update `trunk/`. However, we cannot update the stable version to the new one in step 1, as the tag doesn't exist yet. Obviously, we also cannot leave it at `V.V.V`, so instead, we set it to the previous version.

Reply:
This has a bit of a weird side effect however: A tag's `readme.txt` has the _previous_ version set as its Stable version. See e.g. 9.5.2's `readme.txt`: https://plugins.trac.wordpress.org/browser/gutenberg/tags/9.5.2/readme.txt#L6

Reply:
In a follow-up, I think this could be addressed by:

1. Creating the new tag first.
    - I still have to brush up my svn-fu, but I _think_ this could be done by copying the previous tag to the new one (all remotely?), then checking out the new one locally, and overriding it with the new Gutenberg plugin contents (including the new version). (This assumes that unlike git, svn tags are more like branches, and can be "pushed" to after creating.)
2. Copying/merging/... that new tag to `trunk/`.